src/scrapers/platformen/scraper.py

```diff
@@ -76,10 +84,10 @@ def complete(self, initiative: InitiativeImport):
         """
         pass
 
-    @staticmethod
-    def get(uri):
+    def get(self, uri):
```

Comment:
If the session is part of self, this method is obsolete; would be `self.session.get(url)` instead

Reply:
@wkoot The get method wraps HTTPError. Could get rid of that and do it in the caller.

Reply:
I wasn't aware of the session.headers, thanks very much for that for that. It also automatically merges headers with existing headers like done here:
'response = session.get(uri, headers = self.config.headers)'

Right now I used the session in the get method because it looked more in line with current approach. E.g. there can be different requests per platform. But it is slower creating a new session every time.
What do you think, it's ok? Or should it be moved to a session field of the class PlatformSource?

Reply:
Given the purpose of Session this is not it's intended way of using it: _"The Session object allows you to persist certain parameters across requests. It also persists cookies across all requests made from the Session instance"_ from [this page](https://2.python-requests.org/en/master/user/advanced/#session-objects)

So it belongs in an instance field.